hack/docs/internal/cronjob-tutorial/generate_cronjob.go

```diff
@@ -137,6 +137,10 @@ func (sp *Sample) CodeGen() {
 	cmd = exec.Command("make", "build-installer")
 	_, err = sp.ctx.Run(cmd)
 	hackutils.CheckError("Failed to run make build-installer for cronjob tutorial", err)
+
+	cmd = exec.Command("make", "-C", "../../../../../..", "generate-charts")
```

Comment:
```suggestion
	cmd = exec.Command("kubebuilder", "edit", "--plugins=helm.kubebuilder.io/v1-alpha")
```

Here, we need to use the plugin as an end user would do so.